lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -689,15 +690,169 @@ def fill_in_high_snow_falling_levels(
         snow_level_data[points_not_freezing] = (
             highest_height + orog_data[points_not_freezing])
 
+    def find_extrapolated_falling_level(self, max_wb_integral, gradient,
+                                        intercept, snow_falling_level,
+                                        sea_points):
+        """
+        Find the snow falling level below sea level using the linear
+        extrapolation of the wet bulb temperature integral and update the
+        snow falling level array with these values.
+
+
+        The snow falling level is calculated from finding the point where the
+        integral of wet bulb temperature crosses the falling level threshold.
+
+        In cases where the wet bulb temperature integral has not reached the
+        threshold by time we reach sea level, we can find a fit to the wet bulb
+        temperature profile near the surface, and use this to estimate where
+        the snow falling level would be below sea level.
+
+        The difference between the wet bulb temperature integral at the
+        threshold and the wet bulb integral at the surface is equal to the
+        integral of the wet bulb temperature between sea level and
+        the negative height corresponding to the snow falling level. As we are
+        using a simple linear fit, we can integrate this to find an expression
+        for the extrapolated snow falling level.
+
+        The form of this expression depends on whether the linear fit crosses
+        the x-axis above or below zero.
+
+        If we have our linear fit of the form:
+
+        .. math::
+            {{wet\:bulb\:temperature} = m \\times height + c}
+
+        and let I be the wet bulb temperature integral we have found above
```

Comment:
Can you make "I" here into a maths component so it is formatted to match? Not sure how sphinx is for inline maths.

Reply:
It's really easy. Done